README.md

```diff
@@ -62,70 +100,66 @@ using       ton = unlib::kilo<kilogram>;       // humans don't use megagram
 A quantity combines a [unit](#units) (_mass_), a [scale](#scales) (_kilo_), a specific [value type](#values) (_double_), and an optional [tag](#tags) (_reactive_): 
 
 ```c++
-using kilogram = unlib::quantity<unlib::mass,unlib::kilo,double>. 
+using kilogram = unlib::quantity<unlib::mass,unlib::kilo,double>;
+using     watt = div_quantity_t<joule<V>,second<V>>;
```

Comment:
```suggestion
template<typename V>
using     watt = div_quantity_t<joule<V>,second<V>>;
```

Reply:
Oops. To make the examples simpler, I have always just used a fixed value type.